wikizero.com,"HTTPSConnectionPool(host='wikizero.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='wikizero.com', port=443): Read timed out. (read timeout=2)"
wima.ac.id,"HTTPSConnectionPool(host='wima.ac.id', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdc66df40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='wima.ac.id', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdc5d1820>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
winwin.rs,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
withonline.jp,"HTTPSConnectionPool(host='withonline.jp', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='withonline.jp', port=443): Read timed out. (read timeout=2)"
wixsite.com,"HTTPSConnectionPool(host='wixsite.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdc5b5a30>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='wixsite.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdc630df0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
wolterskluwer.es,"HTTPSConnectionPool(host='wolterskluwer.es', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdc5b5190>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='wolterskluwer.es', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdc683a60>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
woolworths.com.au,"HTTPSConnectionPool(host='www.woolworths.com.au', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.woolworths.com.au', port=443): Read timed out. (read timeout=2)"
wordsense.eu,"HTTPSConnectionPool(host='www.wordsense.eu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.wordsense.eu', port=443): Read timed out. (read timeout=2)"
world.co.jp,"HTTPSConnectionPool(host='world.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdc57ad60>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='world.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdc32fbe0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
www.gdansk.pl,"HTTPSConnectionPool(host='www.gdansk.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.gdansk.pl', port=443): Read timed out. (read timeout=2)"
www.gob.ec,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
xdomain.jp,"HTTPSConnectionPool(host='xdomain.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdc575130>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='xdomain.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdc186f10>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
xn--12c1bik6bbd8ab6hd1b5jc6jta.com,"HTTPSConnectionPool(host='xn--12c1bik6bbd8ab6hd1b5jc6jta.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdc4ced90>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='xn--12c1bik6bbd8ab6hd1b5jc6jta.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdbf8a7f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
xn--22c0bihcc9cwhcxj2ui.com,"HTTPSConnectionPool(host='xn--22c0bihcc9cwhcxj2ui.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='xn--22c0bihcc9cwhcxj2ui.com', port=80): Read timed out. (read timeout=2)"
xn--80acgfbsl1azdqr.xn--p1ai,"HTTPSConnectionPool(host='xn--80acgfbsl1azdqr.xn--p1ai', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='xn--80acgfbsl1azdqr.xn--p1ai', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
xn--lamaana-7za.uy,"HTTPSConnectionPool(host='xn--lamaana-7za.uy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdc44e790>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='xn--lamaana-7za.uy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdc606a30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
xn--mgbkbq.xn--mgberp4a5d4ar,"HTTPSConnectionPool(host='xn--mgbkbq.xn--mgberp4a5d4ar', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'xn--mgbkbq.xn--mgberp4a5d4ar' doesn't match either of '*.haraj.com.sa', 'haraj.com', 'haraj.com.sa', '*.haraj.com'"")))","HTTPSConnectionPool(host='xn--mgbkbq.xn--mgberp4a5d4ar', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'xn--mgbkbq.xn--mgberp4a5d4ar' doesn't match either of '*.haraj.com.sa', 'haraj.com', 'haraj.com.sa', '*.haraj.com'"")))"
xn--titnjaa-o6a36e.hr,"HTTPSConnectionPool(host='xn--titnjaa-o6a36e.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='xn--titnjaa-o6a36e.hr', port=443): Read timed out. (read timeout=2)"
xn--y8jdm4g.com,"HTTPSConnectionPool(host='xn--y8jdm4g.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdbf69940>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='xn--y8jdm4g.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdc379610>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
xoglasi.com,"HTTPSConnectionPool(host='xoglasi.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='xoglasi.com', port=443): Read timed out. (read timeout=2)"
xuite.net,"HTTPSConnectionPool(host='xuite.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='xuite.net', port=443): Read timed out. (read timeout=2)"
y-lehti.fi,"HTTPSConnectionPool(host='www.y-lehti.fi', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.y-lehti.fi', port=443): Read timed out. (read timeout=2)"
yamada-denkiweb.com,"HTTPSConnectionPool(host='yamada-denkiweb.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdc0492b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='yamada-denkiweb.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdc475e20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
yamaguchi.lg.jp,"HTTPSConnectionPool(host='yamaguchi.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdc0d1ee0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='yamaguchi.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdbf8ce20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
yamano-music.co.jp,"HTTPSConnectionPool(host='yamano-music.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdc348340>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='yamano-music.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdc6061c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
yaom.ru,"HTTPSConnectionPool(host='yaom.ru', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdc33dc70>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='yaom.ru', port=80): Read timed out. (read timeout=2)"
yatra.com,"HTTPSConnectionPool(host='www.yatra.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.yatra.com', port=443): Read timed out. (read timeout=2)"
yavitrina.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPConnectionPool(host='yavitrina.ru', port=80): Read timed out. (read timeout=2)"
ybz.org.il,"HTTPSConnectionPool(host='www.ybz.org.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.ybz.org.il', port=443): Read timed out. (read timeout=2)"
yeditepehastanesi.com.tr,"HTTPSConnectionPool(host='yeditepehastanesi.com.tr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdbce8310>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='yeditepehastanesi.com.tr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdc033bb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
yellowpages.co.za,"HTTPSConnectionPool(host='yellowpages.co.za', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.yellowpages.co.za', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
yellowpages.com,"HTTPSConnectionPool(host='yellowpages.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f8bdbca25b0>, 'Connection to yellowpages.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='yellowpages.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f8bdc0ba520>, 'Connection to yellowpages.com timed out. (connect timeout=2)'))"
yellowpageskenya.com,"HTTPSConnectionPool(host='yellowpageskenya.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='yellowpageskenya.com', port=443): Read timed out. (read timeout=2)"
yellshops.com,"HTTPSConnectionPool(host='yellshops.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='yellshops.com', port=443): Read timed out. (read timeout=2)"
yerevanmobile.am,"HTTPSConnectionPool(host='yerevanmobile.am', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='yerevanmobile.am', port=443): Read timed out. (read timeout=2)"
yhb.org.il,"HTTPSConnectionPool(host='yhb.org.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='yhb.org.il', port=443): Read timed out. (read timeout=2)"
ymall.jp,"HTTPSConnectionPool(host='ymall.jp', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ymall.jp', port=443): Read timed out. (read timeout=2)"
yodobashi.com,"HTTPSConnectionPool(host='yodobashi.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdbc4b700>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='yodobashi.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdbc64970>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
yokohama.lg.jp,"HTTPSConnectionPool(host='yokohama.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdbc64d90>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='yokohama.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdc077ca0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
yoys.ae,"HTTPSConnectionPool(host='www.yoys.ae', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.yoys.ae', port=443): Read timed out. (read timeout=2)"
yoys.sg,"HTTPSConnectionPool(host='yoys.sg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'yoys.sg' doesn't match either of 'www.yoys.ae', 'yoys.ae'"")))","HTTPSConnectionPool(host='www.yoys.sg', port=443): Read timed out. (read timeout=2)"
ypfb.gob.bo,"HTTPSConnectionPool(host='ypfb.gob.bo', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f8bdcdb5c10>, 'Connection to ypfb.gob.bo timed out. (connect timeout=2)'))","HTTPConnectionPool(host='ypfb.gob.bo', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f8bdc2be2b0>, 'Connection to ypfb.gob.bo timed out. (connect timeout=2)'))"
yubox.com,"HTTPSConnectionPool(host='yubox.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='yubox.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
zacimbe.si,"HTTPSConnectionPool(host='zacimbe.si', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='zacimbe.si', port=443): Read timed out. (read timeout=2)"
zagonka.net,"HTTPSConnectionPool(host='zagonka.net', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdbd1c700>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='zagonka.net', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdbb5b1c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
zalando.at,"HTTPSConnectionPool(host='www.zalando.at', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.at', port=443): Read timed out. (read timeout=2)"
zalando.be,"HTTPSConnectionPool(host='www.zalando.be', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.be', port=443): Read timed out. (read timeout=2)"
zalando.ch,"HTTPSConnectionPool(host='www.zalando.ch', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.ch', port=443): Read timed out. (read timeout=2)"
zalando.co.uk,"HTTPSConnectionPool(host='www.zalando.co.uk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.co.uk', port=443): Read timed out. (read timeout=2)"
zalando.cz,"HTTPSConnectionPool(host='www.zalando.cz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.cz', port=443): Read timed out. (read timeout=2)"
zalando.de,"HTTPSConnectionPool(host='www.zalando.de', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.de', port=443): Read timed out. (read timeout=2)"
zalando.dk,"HTTPSConnectionPool(host='www.zalando.dk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.dk', port=443): Read timed out. (read timeout=2)"
zalando.es,"HTTPSConnectionPool(host='www.zalando.es', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.es', port=443): Read timed out. (read timeout=2)"
zalando.fi,"HTTPSConnectionPool(host='www.zalando.fi', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.fi', port=443): Read timed out. (read timeout=2)"
zalando.fr,"HTTPSConnectionPool(host='www.zalando.fr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.fr', port=443): Read timed out. (read timeout=2)"
zalando.ie,"HTTPSConnectionPool(host='www.zalando.ie', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.ie', port=443): Read timed out. (read timeout=2)"
zalando.it,"HTTPSConnectionPool(host='www.zalando.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.it', port=443): Read timed out. (read timeout=2)"
zalando.nl,"HTTPSConnectionPool(host='www.zalando.nl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.nl', port=443): Read timed out. (read timeout=2)"
zalando.no,"HTTPSConnectionPool(host='www.zalando.no', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.no', port=443): Read timed out. (read timeout=2)"
zalando.pl,"HTTPSConnectionPool(host='www.zalando.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.pl', port=443): Read timed out. (read timeout=2)"
zalando.se,"HTTPSConnectionPool(host='www.zalando.se', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zalando.se', port=443): Read timed out. (read timeout=2)"
zalora.co.id,"HTTPSConnectionPool(host='www.zalora.co.id', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.zalora.co.id', port=80): Read timed out. (read timeout=2)"
zalora.com.my,"HTTPSConnectionPool(host='www.zalora.com.my', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.zalora.com.my', port=80): Read timed out. (read timeout=2)"
zalora.sg,"HTTPSConnectionPool(host='www.zalora.sg', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.zalora.sg', port=80): Read timed out. (read timeout=2)"
zamanalwsl.net,"HTTPSConnectionPool(host='zamanalwsl.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='zamanalwsl.net', port=443): Read timed out. (read timeout=2)"
zaobao.com.sg,"('Received response with content-encoding: gzip, but failed to decode it.', error('Error -3 while decompressing data: incorrect header check'))","('Received response with content-encoding: gzip, but failed to decode it.', error('Error -3 while decompressing data: incorrect header check'))"
zapmarket.co.il,"HTTPSConnectionPool(host='zapmarket.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='zapmarket.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
zdravniki-zobozdravniki.net,"HTTPSConnectionPool(host='www.zdravniki-zobozdravniki.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zdravniki-zobozdravniki.net', port=443): Read timed out. (read timeout=2)"
zilinskazupa.sk,"HTTPSConnectionPool(host='zilinskazupa.sk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8bdc630ee0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='zilinskazupa.sk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8bdbe31730>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
zimmer4me.co.il,"HTTPSConnectionPool(host='www.zimmer4me.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.zimmer4me.co.il', port=443): Read timed out. (read timeout=2)"
ziveobce.cz,"HTTPSConnectionPool(host='ziveobce.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ziveobce.cz' doesn't match either of '*.zivefirmy.cz', 'zivefirmy.cz'"")))","HTTPSConnectionPool(host='ziveobce.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ziveobce.cz' doesn't match either of '*.zivefirmy.cz', 'zivefirmy.cz'"")))"
